extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -185,7 +185,7 @@ static function renderDefaultform( &$parser ) {
 		// It's not a category - display nothing.
 	}
 
-	static function renderFormLink ( &$parser ) {
+	static function renderFormLink ( $parser ) {
 		$params = func_get_args();
 		array_shift( $params ); // We don't need the parser.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)